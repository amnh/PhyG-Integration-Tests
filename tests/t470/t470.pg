--softwired, nopenalty, netmove,  atrandom, steepest
set(seed:1634561640)
set(outgroup:T0)
set(graphtype:softwired)
set(graphfactor:nopenalty) 
read(fasta:"Mut2_1_sars-cov_reduced.fasta")
read(fasta:"Mut2_2_sars-cov_reduced.fasta")
read(newick:"../t467/t467.tre")
refine(netmove, atrandom, steepest)
report("t470.csv", data, overwrite)
report("t470.tre", newick, graphs, overwrite)
report("t470_gv.tre", dotpdf, graphs, overwrite)
