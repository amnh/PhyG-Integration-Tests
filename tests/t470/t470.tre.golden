(AlbanianTosk:0.0,((Irish:73.5,((ArmenianWestern:155.0,((((Yiddish:109.3,((Icelandic:196.3,KolschColognian:133.0)HTU25:2.0)#HTU40:2.0)HTU24:108.0,(Danish:0.0)#HTU26:0.0)HTU23:19.0,((AncientIonicGreek:161.2,((Assamese:77.7,SanskritVedic:51.0)HTU30:0.0)#HTU29:0.0)HTU28:0.0)#HTU27:0.0)HTU22:57.9,(EasternFrisianSaterland:57.9,((Faroese:72.0,#HTU29:2.0)HTU33:0.0,((Scots:69.0,#HTU40:0.0)HTU39:0.0,#HTU27:0.0)HTU34:26.0)HTU32:41.4)HTU31:48.0)HTU21:85.4)HTU20:26.7,((Cornish:95.5,#HTU26:0.0)HTU38:0.0)#HTU35:0.0)HTU19:0.0)HTU18:43.1,(Manx:50.1,(OldPrussian:67.4,#HTU35:0.0)HTU37:29.0)HTU36:57.0)HTU17:0.0)HTU16[3540.0];
(AlbanianTosk:0.0,(((((((Yiddish:111.3,(Icelandic:195.3,KolschColognian:133.0)HTU24:82.6)HTU23:109.0,(Danish:0.0)#HTU25:0.0)HTU22:19.0,((AncientIonicGreek:160.2,((Assamese:77.7,SanskritVedic:51.0)HTU29:0.0)#HTU28:0.0)HTU27:0.0)#HTU26:0.0)HTU21:59.9,(EasternFrisianSaterland:57.9,((Faroese:74.0,#HTU28:2.0)HTU32:0.0,(Scots:69.0,#HTU26:0.0)HTU33:26.0)HTU31:43.4)HTU30:46.0)HTU20:81.4,(ArmenianWestern:155.0,#HTU40:0.0)HTU39:0.0)HTU19:24.7,(Irish:0.0)#HTU34:0.0)HTU18:42.1,(Manx:45.1,((OldPrussian:66.4,((Cornish:96.5,#HTU25:0.0)HTU38:0.0)#HTU40:0.0)HTU37:28.0,#HTU34:0.0)HTU36:4.0)HTU35:58.0)HTU17:0.0)HTU16[3538.0];
(AlbanianTosk:0.0,((Irish:71.6,(Manx:46.1,(((OldPrussian:79.4,#HTU38:0.0)HTU37:24.0,(AncientIonicGreek:0.0)#HTU22:0.0)HTU21:24.0,((Cornish:59.5,(Scots:0.0)#HTU25:0.0)HTU24:0.0)#HTU23:0.0)HTU20:17.0)HTU19:31.0)HTU18:60.7,((((Yiddish:114.2,((Icelandic:193.0,KolschColognian:137.3)HTU30:76.0,#HTU40:0.0)HTU39:6.0)HTU29:97.0,((Assamese:72.7,SanskritVedic:47.0)HTU32:111.0,#HTU22:0.0)HTU31:22.0)HTU28:74.0,(((Faroese:78.0,(EasternFrisianSaterland:0.0)#HTU40:0.0)HTU35:25.0,#HTU25:0.0)HTU34:73.3,(Danish:0.0)#HTU38:0.0)HTU33:11.1)HTU27:93.1,(ArmenianWestern:143.1,#HTU23:0.0)HTU36:0.0)HTU26:35.0)HTU17:1.0)HTU16[3530.4];
(AlbanianTosk:0.0,((Irish:74.5,(Manx:52.1,((OldPrussian:78.4,(AncientIonicGreek:0.0)#HTU22:0.0)HTU21:4.0,((Cornish:98.5,(Danish:0.0)#HTU40:0.0)HTU24:0.0)#HTU23:0.0)HTU20:19.0)HTU19:34.1)HTU18:60.7,((((((Assamese:74.7,SanskritVedic:49.0)HTU29:119.0,#HTU22:0.0)HTU28:22.1,#HTU40:0.0)HTU39:3.0,((Yiddish:111.3,((Icelandic:196.3,KolschColognian:134.0)HTU32:2.0)#HTU38:2.0)HTU31:17.0)#HTU30:17.0)HTU27:90.1,(Scots:61.0,(Faroese:78.0,(EasternFrisianSaterland:89.3,#HTU38:0.0)HTU37:0.0)HTU34:30.0)HTU33:59.2)HTU26:72.2,((ArmenianWestern:136.0,#HTU30:0.0)HTU36:0.0,#HTU23:0.0)HTU35:0.0)HTU25:38.0)HTU17:0.0)HTU16[3523.6];
(AlbanianTosk:0.0,((((Scots:63.0,(((EasternFrisianSaterland:91.3,Faroese:78.0)HTU23:24.0,(Irish:0.0)#HTU24:0.0)HTU22:0.0)#HTU21:0.0)HTU20:58.2,((AncientIonicGreek:166.3,((Assamese:73.6,(SanskritVedic:45.1,(Yiddish:0.0)#HTU29:0.0)HTU28:0.0)HTU27:111.0,#HTU40:0.0)HTU39:2.0)HTU26:47.0,((KolschColognian:134.0,(Icelandic:195.3,#HTU21:0.0)HTU32:0.0)HTU31:76.6,#HTU29:0.0)HTU30:113.0)HTU25:91.1)HTU19:76.2,(ArmenianWestern:149.0,(Cornish:0.0)#HTU34:0.0)HTU33:0.0)HTU18:33.0,(((Manx:41.1,(OldPrussian:66.4,(Danish:0.0)#HTU40:0.0)HTU38:31.0)HTU37:36.1,#HTU34:0.0)HTU36:29.0,#HTU24:0.0)HTU35:58.7)HTU17:1.0)HTU16[3520.5];
(AlbanianTosk:0.0,(((((((Yiddish:112.3,((Icelandic:195.3,KolschColognian:134.0)HTU24:2.0)#HTU40:2.0)HTU23:109.0,(Danish:0.0)#HTU25:0.0)HTU22:18.0,((AncientIonicGreek:160.2,(Assamese:73.7,SanskritVedic:50.0)HTU28:119.0)HTU27:0.0)#HTU26:0.0)HTU21:59.9,(EasternFrisianSaterland:58.9,(Faroese:73.0,((Scots:72.0,#HTU40:0.0)HTU39:0.0,#HTU26:0.0)HTU31:26.0)HTU30:43.4)HTU29:46.0)HTU20:81.4,(ArmenianWestern:155.0,#HTU38:0.0)HTU37:0.0)HTU19:25.7,(Irish:0.0)#HTU32:0.0)HTU18:42.1,(Manx:45.1,((OldPrussian:66.4,((Cornish:96.5,#HTU25:0.0)HTU36:0.0)#HTU38:0.0)HTU35:28.0,#HTU32:0.0)HTU34:4.0)HTU33:58.0)HTU17:0.0)HTU16[3520.0];
(AlbanianTosk:0.0,((Irish:76.5,(Manx:39.1,(((OldPrussian:78.4,(AncientIonicGreek:0.0)#HTU22:0.0)HTU21:6.0,#HTU40:0.0)HTU39:38.0,(Cornish:0.0)#HTU23:0.0)HTU20:15.0)HTU19:36.1)HTU18:59.7,((((((Assamese:74.7,SanskritVedic:48.0)HTU28:119.0,#HTU22:0.0)HTU27:22.1,(Danish:0.0)#HTU40:0.0)HTU38:3.0,((Yiddish:111.3,((Icelandic:196.3,KolschColognian:134.0)HTU31:2.0)#HTU37:2.0)HTU30:17.0)#HTU29:17.0)HTU26:91.1,(Scots:61.0,(Faroese:77.0,(EasternFrisianSaterland:88.3,#HTU37:0.0)HTU36:0.0)HTU33:30.0)HTU32:59.2)HTU25:71.2,((ArmenianWestern:137.0,#HTU29:0.0)HTU35:0.0,#HTU23:0.0)HTU34:0.0)HTU24:35.0)HTU17:1.0)HTU16[3507.6];
(AlbanianTosk:0.0,((((Scots:68.0,((Faroese:76.0,(EasternFrisianSaterland:0.0)#HTU40:0.0)HTU22:20.0,(Irish:0.0)#HTU23:0.0)HTU21:0.0)HTU20:56.2,(((Icelandic:194.3,KolschColognian:136.0)HTU30:75.0,#HTU28:0.0)HTU29:114.0,((AncientIonicGreek:165.3,((Assamese:73.6,(SanskritVedic:45.1,(Yiddish:0.0)#HTU28:0.0)HTU27:0.0)HTU26:111.0,#HTU38:0.0)HTU37:3.0)HTU25:49.6,#HTU40:0.0)HTU39:1.0)HTU24:88.5)HTU19:75.2,(ArmenianWestern:149.0,(Cornish:0.0)#HTU32:0.0)HTU31:0.0)HTU18:34.0,(((Manx:41.1,(OldPrussian:66.4,(Danish:0.0)#HTU38:0.0)HTU36:31.0)HTU35:36.1,#HTU32:0.0)HTU34:29.0,#HTU23:0.0)HTU33:58.7)HTU17:1.0)HTU16[3503.4];
(AlbanianTosk:0.0,((Irish:75.5,((((AncientIonicGreek:159.2,(SanskritVedic:49.0,(Assamese:0.0)#HTU40:0.0)HTU26:108.0)HTU25:52.1,(((Yiddish:105.3,((Icelandic:196.3,KolschColognian:136.0)HTU24:2.0)#HTU31:2.0)HTU23:103.9,(Danish:0.0)#HTU33:0.0)HTU22:4.0,(EasternFrisianSaterland:0.0)#HTU38:0.0)HTU36:19.9)HTU21:85.1,((Faroese:67.0,#HTU31:0.0)HTU30:0.0,((Scots:87.0,#HTU38:0.0)HTU37:21.0,#HTU40:0.0)HTU39:1.0)HTU27:56.2)HTU20:79.2,(ArmenianWestern:149.0,(Cornish:0.0)#HTU35:0.0)HTU29:0.0)HTU19:21.7)HTU18:41.1,((OldPrussian:76.4,#HTU33:0.0)HTU32:22.0,(Manx:51.1,#HTU35:0.0)HTU34:26.0)HTU28:60.0)HTU17:0.0)HTU16[3485.7];
(AlbanianTosk:0.0,((Irish:75.5,((((AncientIonicGreek:140.3,((Assamese:72.6,SanskritVedic:48.1)HTU26:0.0)#HTU40:0.0)HTU25:48.0,((((Yiddish:104.3,((Icelandic:197.3,KolschColognian:135.0)HTU24:2.0)#HTU31:2.0)HTU23:72.0,#HTU40:0.0)HTU39:35.9,(Danish:0.0)#HTU33:0.0)HTU22:2.0,(EasternFrisianSaterland:0.0)#HTU38:0.0)HTU36:40.9)HTU21:85.1,((Faroese:58.0,#HTU31:0.0)HTU30:0.0,(Scots:95.0,#HTU38:0.0)HTU37:21.0)HTU27:43.2)HTU20:81.2,(ArmenianWestern:149.0,(Cornish:0.0)#HTU35:0.0)HTU29:0.0)HTU19:21.7)HTU18:41.1,((OldPrussian:76.4,#HTU33:0.0)HTU32:22.0,(Manx:51.1,#HTU35:0.0)HTU34:26.0)HTU28:60.0)HTU17:0.0)HTU16[3472.7];
